<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Frames</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="framesSpinBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cameraOnCheckBox">
     <property name="text">
      <string>camera on</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QComboBox" name="camerasComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
